package.json

```diff
@@ -198,7 +198,7 @@
     "size-limit": "^0.21.0",
     "styled-components": "^4.1.1",
     "tslint": "^5.11.0",
-    "typescript": "3.2.2",
+    "typescript": "^3.4.4",
```

Comment:
I would rather revisit this separately. TypeScript doesn't follow SemVer which means that this might require unrelated changes.

It also means that we could use 3.4 features in our declarations which essentially means consumers need to upgrade to 3.4 as well.

Reply:
I was unable to get the `flexWrap` CSS property to work without it (as in this was the first thing i tried, and it worked so I didn't try anything else).

Reply:
The issue that required the update is this one https://github.com/Microsoft/TypeScript/pull/30215
So using a lower version doesn't seem possible